docs/guides/usage.md

```diff
@@ -0,0 +1,221 @@
+# Execution of pipelines
+
+ZARP consists of three different pipelines. The main pipeline processes the data, the second allows you to download the sequencing libraries from the Sequence Read Archive (SRA), and the third populates a file with the samples and determines sample specific parameters.
```

Comment:
I wouldn't say that ZARP consists of three different pipelines. Rather, ZARP comes with two auxiliary workflows.

Perhaps write something like this:

> Next to the ZARP workflow for RNA-Seq analysis, this project comes with two auxiliary workflows for fetching samples from the Sequence Read Archive and populating a sparse sample table with inferred sample metadata. This section describes how to run each of these workflows.

Reply:
Done